wsgi.py

```diff
@@ -1,55 +1,59 @@
 import os
 import logging
+import io
+import tarfile
+import requests
 
 from flask import Flask, jsonify, request
 from flask.logging import default_handler
 
-import s3
-import producer
-
 application = Flask(__name__)  # noqa
 
 # Set up logging
 ROOT_LOGGER = logging.getLogger()
 ROOT_LOGGER.setLevel(application.logger.level)
 ROOT_LOGGER.addHandler(default_handler)
 
-# Kafka message bus
-SERVER = os.environ.get('KAFKA_SERVER')
-TOPIC = os.environ.get('KAFKA_TOPIC')
-
-# S3 credentials
-AWS_KEY = os.environ.get('AWS_ACCESS_KEY_ID')
-AWS_SECRET = os.environ.get('AWS_SECRET_ACCESS_KEY')
-AWS_BUCKET = os.environ.get('AWS_S3_BUCKET_NAME')
+# Upload Service
+UPLOAD_SERVICE = os.environ.get('UPLOAD_SERVICE')
```

Comment:
What this stands for? Is that a `host:port`? Let's rather call it `UPLOAD_SERVICE_HOST` then, what do you think?

Reply:
It's actually the Upload Service endpoint - `upload-service:8080/api/v1/upload`, hence I have renamed the env variable to `UPLOAD_SERVICE_ENDPOINT`